nnpdf_data/nnpdf_data/filter_utils/star/jet_dijet_2009_200gev.py

```diff
@@ -0,0 +1,319 @@
+
+        else:
+            print("ERROR: Unknown table number detected! Check input files.")
+
+    df = pd.concat([df3, df4], ignore_index=True)
+    df["pol"] = POL_UNC * df["ALL"]
+    df["lumi_ue"] = LUMI_UNC
+
+    for i in range(len(df)):
+        df.loc[i, "stat"] = df.loc[i, "stat_max"]
+        df.loc[i, "sys"] = df.loc[i, "sys_max"]
+    return df, dfc
+
+
+def read_2jet_data(topology):
+    fname = f"../../new_commondata/STAR_2009_2JET_{topology}_200GEV/rawdata/Table.yaml"
+    df = pd.DataFrame()
+    with open(fname, "r") as file:
+        data = yaml.safe_load(file)
+
+    if topology in ["A", "B", "C"]:
+        Msub = data["independent_variables"][0]["values"]
+        Gsub = data["dependent_variables"][0]["values"]
+
+        for i in range(len(Msub)):
+
+            df = pd.concat(
+                [
+                    df,
+                    pd.DataFrame(
+                        {
+                            "m_low": [Msub[i]["low"]],
+                            "m_high": [Msub[i]["high"]],
+                            "ALL": [Gsub[i]["value"]],
+                            "stat": [Gsub[i]["errors"][0]["symerror"]],
+                            "sys": [Gsub[i]["errors"][1]["symerror"]],
+                            "pol": [POL_UNC * Gsub[i]["value"]],
+                            "lumi_ue": [LUMI_UNC],
+                        }
+                    ),
+                ],
+                ignore_index=True,
+            )
+
+        df["m"] = (df["m_low"] + df["m_high"]) / 2
+        df["sqrts"] = 200
+
+    else:
+        Msub = data["dependent_variables"][0]["values"]
+        Gsub = data["dependent_variables"][1]["values"]
+
+        for i in range(len(Msub)):
+
+            df = pd.concat(
+                [
+                    df,
+                    pd.DataFrame(
+                        {
+                            "m": [Msub[i]["value"]],
+                            "ALL": [Gsub[i]["value"]],
+                            "stat": [Gsub[i]["errors"][0]["symerror"]],
+                            "sys": [Gsub[i]["errors"][1]["symerror"]],
+                            "pol": [POL_UNC * Gsub[i]["value"]],
+                            "lumi_ue": [LUMI_UNC],
+                        }
+                    ),
+                ],
+                ignore_index=True,
+            )
+        df["sqrts"] = 200
+
+    return df
+
+
+def write_1jet_data(df, art_sys):
+    STORE_PATH = "../../new_commondata/STAR_2009_1JET_200GEV/"
+
+    # Write central data
+    data_central_yaml = {"data_central": list(df["ALL"])}
+    with open(STORE_PATH + "data.yaml", "w", encoding="utf-8") as file:
+        yaml.dump(data_central_yaml, file)
+
+    # Write kin file
+    kin = []
+    for i in range(len(df)):
+        kin_value = {
+            "pT": {"min": None, "mid": float(df.loc[i, "pT"]), "max": None},
+            "sqrts": {"min": None, "mid": float(df.loc[i, "sqrts"]), "max": None},
+            "eta": {
```

Comment:
```suggestion
            "abs_eta": {
```
and similarly in the metadata (if not correct).